zeppelin-server/src/test/java/org/apache/zeppelin/rest/message/UpdateInterpreterSettingRequestTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.rest.message;
+
+import org.apache.zeppelin.dep.Dependency;
+import org.apache.zeppelin.interpreter.InterpreterOption;
+import org.apache.zeppelin.interpreter.InterpreterProperty;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Unit test class for the UpdateInterpreterSettingRequest.
+ * This class tests the behavior of UpdateInterpreterSettingRequest methods,
+ * especially focusing on property type handling and value conversions.
+ */
+class UpdateInterpreterSettingRequestTest {
+
+  private Map<String, InterpreterProperty> properties;
+  private List<Dependency> dependencies;
+  private InterpreterOption option;
+  private UpdateInterpreterSettingRequest request;
+
+  /**
+   * Setup method that initializes test fixtures before each test.
+   * Mocks dependencies and option objects to isolate UpdateInterpreterSettingRequest behavior.
+   */
+  @BeforeEach
+  void setUp() {
+    properties = new HashMap<>();
+    dependencies = mock(List.class);
```

Comment:
I get the following warning in my IDE.
![grafik](https://github.com/user-attachments/assets/dbc19d3f-89c0-4005-bc05-6983483af38c)

To prevent this, we can also pass a normal empty list.
```suggestion
    dependencies = new LinkedList<>();
```